examples/federated_learning/yolov5_coco128_mistnet/interface.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from sedna.algorithms.aggregation import MistNet
+from sedna.algorithms.client_choose import SimpleClientChoose
+from sedna.common.config import Context
+from sedna.core.federated_learning import FederatedLearning
+
+simple_chooser = SimpleClientChoose(per_round=1)
+
+# It has been determined that mistnet is required here.
+mistnet = MistNet(cut_layer=Context.get_parameters("cut_layer"),
+                  epsilon=Context.get_parameters("epsilon"))
```

Comment:
Seems these values is not define in `train worker`

Reply:
```train worker``` uses ```interface.mistnet```.